src/App.jsx

```diff
@@ -1,11 +1,66 @@
 import { useState } from "react";
 import "./App.css";
-
+import {AiFillPlusCircle} from "react-icons/ai"
+import Todo from "./components/Todo";
+import { nanoid } from "nanoid";
 
 const App =()=> {
+  const [inputValue, setInputValue] = useState("");
+  const [todo, setTodo] = useState([])
+
+  const handleInput =(event) => {
+    setInputValue(prevInput=> (prevInput, event.target.value) )
+  }
+
+  const handleDelete =(id) =>{
+    setTodo(prevTodo => prevTodo.filter((obj)=> obj.props.id != id))
+  }
+
+  const handleEnter = (event) => {
+    if(event.key === "Enter"){
+      addTodo()
+    }
+  }
+
+  const toggleCompleted = (id) => {
+    setTodo(prevTodo => {
+      return prevTodo.map(obj => {
+        if (obj.props.id === id) {
+          return {
+            ...obj,
+            props: {
+              ...obj.props,
+              complete: !obj.props.complete
+            }
+          }
+        }
+        return obj
+      })
+    })
+  }
+    
+  const addTodo = () =>{
+    if(inputValue.trim() === ""){
+      return
+    }  
+    setInputValue("")
+    return(
```

Comment:
I think you remove unnecessary return statement in addTodo() function and find a way of using one

Reply:
I used that return to exit out of the function if the condition is true, I don't see how else I can do it